### DISCUSSION TITLE
UIKit framework?

### DISCUSSION THREAD

**Discussion by derkolarsky** [automadcms/discussions#28] [Q&A]
Hi @marcantondahmen,

Do I get this right, that the standard theme files (I'm using the 1.10.9) are using the framework from UIKit? If so: awesome! May I ask, how much of all the available components that are listed on getuikit.org are included in the theme?
I'm trying to get the uk-inverse to work for the navbar…

Thanks for any feedback in advance.

Ralf

**Comment by marcantondahmen** [comment]
Hi, yes the theme is based on UIkit. However, it is based on UIkit v2. Also it is quite customized and not all features are supported. But it should be possible to cover more UIkit features with some small modifications.
